src/main_controller.ts

```diff
@@ -257,10 +257,15 @@ export class MainController implements vscode.Disposable {
             }
             targetRange = doc.validateRange(targetRange);
             editor.selections = [new vscode.Selection(targetRange.start, targetRange.end)];
+            // This is an arbitrary action from VSCode, we can't know the return type
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
             const res = await actions.run(action, ...(options.args || []));
```

Comment:
I really think this kind of comment is redundant.....

Reply:
It's a bit repetitive, sure. I did it just to justify the lint disabling, but if you would prefer it gone I can.